jerry-core/ecma/base/ecma-globals.h

```diff
@@ -656,6 +656,11 @@ typedef enum
  */
 #define ECMA_OBJECT_FLAG_EXTENSIBLE 0x20
 
+/**
+ * Catch clause.
+ */
+#define ECMA_OBJECT_FLAG_CATCH_CLAUSE 0x20
```

Comment:
I would call it local scope, since we might add let/const at some day and they would be marked this flag as well.